osgar/drivers/logzeromq.py

```diff
@@ -76,22 +76,75 @@ def run_output(self):
         socket.close()
         context.term()
 
+    
+    class ReqRepWorker(Thread):
```

Comment:
can we move this "worker pool" from OSGAR to ROS (or rosproxy handler to be more precise)? I.e. there would be only I/O line on OSGAR side and optionally multiple Threads on rospy to handle multiple requests??

Reply:
I don't immediately see that. For starters, the processing of the request ends when 'response' is published into OSGAR bus. rospy is not aware of and a node to OSGAR. Alternatively, if you wait for response from rospy in logzeromq without using threads, you're back at serialized blocking calls where you started from.